Layout and Positioning Pull Request

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+.flex-container {
```

Comment:
This is okay, but remember that best practice is to put your CSS in a separate file and include via a link tag.